cinderback.py

```diff
@@ -780,7 +785,7 @@ def get_client(self, tenant_id, keep_tenant=True):
         """Return a client for requested tenant"""
         # If we are the original tenant of the volume
         if (not keep_tenant or
-                self.client.client.auth_ref['token']['tenant']['id']
+                self.client.client.auth_ref._data['token']['project']['id']
```

Comment:
I'd rather avoid using internal attributes, could we use `self.client.client.auth_ref.tenant_id` which is a property that will return 'tenant' or 'project' depending on the version in use?

Using `tenant_id` property instead of `project_id` seems more intuitive when comparing with tenant_id parameter.